guardian/lib/slack/model/slack_section_block.dart

```diff
@@ -0,0 +1,71 @@
+import 'package:equatable/equatable.dart';
+import 'package:guardian/slack/model/slack_text_object.dart';
+
+/// A class representing Section Block element
+/// (https://api.slack.com/reference/block-kit/blocks#section)
+class SlackSectionBlock extends Equatable {
+  /// The text for the block, in the form of a [SlackTextObject].
+  ///
+  /// Maximum length for the text in this field is 3000 characters.
+  final SlackTextObject text;
+
+  /// An array of text objects. Any text objects included with fields will be
+  /// rendered in a compact format that allows for 2 columns of side-by-side
+  /// text.
+  ///
+  /// Maximum number of items is 10. Maximum length for the
+  /// [SlackTextObject.text] in each item is 2000 characters.
```

Comment:
should we add some validation code (asserts or something else) to make sure that those rules are followed?